### ISSUE TITLE
Module out of date

### ISSUE THREAD

**Issue by YabaiKai** [fship7/asustor-nordlynx#1]
I'd like to use this, but I think the module for wireguard is now out of date since Asustor updated to kernel 5.13.x with ADM 4.1

**Comment by fship7** [comment]
Yes unfortunately it's out of date. Another problem is that kernel 5.13
should have native wireguard support which would make this repo
unnecessary, but for some reason asustor decided to deliberately remove
wireguard from 5.13.x. This means the whole kernel needs to be recompiled
with the wireguard module enabled. I'm not exactly sure how to even do it.
Sorry.

- fship7
